### PR TITLE
Example of a Lagom Scala project to run outside of ConductR

### PR DESCRIPTION
**Pull request by gupta-himanshu**
A basic example of building a Lagom project (in Scala) to run outside of the Lightbend Production Suite (aka ConductR).

### CODE REVIEW

**Comment by gupta-himanshu** [comment]
@TimMoore Any feedback?

**Comment by TimMoore** [comment]
Sorry @gupta-himanshu, I missed this. This repository is just a quick experiment I threw together, and I haven't been paying much attention to it or publicizing it.

But I'm happy to include the link to your example. There's just a small typo in the link text. Thanks!

File: README.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ This runs each of the services on the configured port, as single-node clusters.
 
 You can press control-C to exit the services, and run `lagomCassandraStop` in the initial sbt console to stop Cassandra (or simply exit sbt).
 
+## Steps for lagom-scala project
+
+For those looking for a basic example of building a Lagom project, in Scala, to run outside of the Lightbend Production Suite (aka ConductR) follow this link - [lafom-scala-sbt-standalone](https://github.com/knoldus/lagom-scala-sbt-standalone).
```
Comment (TimMoore) [inline README.md]: Small typo: "lafom" --> "lagom"

**Comment by gupta-himanshu** [comment]
Thanks for the feedback @TimMoore. I have corrected the typo and updated the PR. Please let me know if anything else needs to be changed.

**Review by TimMoore — COMMENTED** [review]
Thanks, @gupta-himanshu! Sorry about the delay.

**Pull request merged**